.vscode/settings.json

```diff
@@ -6,7 +6,6 @@
   "editor.tabSize": 2,
   "editor.insertSpaces": true,
   "editor.detectIndentation": false,
-  "editor.formatOnSave": true,
```

Comment:
Why did you remove this?

Reply:
With my config, I end up making es-lint/ts-lint on save because it formats. For instance, spaces between function name and parentheses. Do we want this though?

Reply:
We want auto formating, like Prettier, on save yes?